NDB.Covid19/NDB.Covid19/Utils/DateUtils.cs

```diff
@@ -16,8 +16,9 @@ public static string GetDateFromDateTime(this DateTime? date, string dateFormat)
                 DateTime dateTime = (DateTime)date;
                 string appLanguage = LocalPreferencesHelper.GetAppLanguage();
                 CultureInfo selectedCulture = CultureInfo.GetCultureInfo(appLanguage);
-                CultureInfo defaultCulture = CultureInfo.GetCultureInfo(Conf.DEFAULT_LANGUAGE);
-                bool shouldUseDefaultCulture = appLanguage == "ar" || appLanguage == "ur" || appLanguage == "ti";
+                // Due to a bug in C# string representation in nb culture, nn must be used
+                CultureInfo defaultCalendarCulture = CultureInfo.GetCultureInfo("nn");
```

Comment:
Why didn't you change it in the config file in `Conf.DEFAULT_LANGUAGE` as it was done before?

Reply:
It doesn't look like the default culture is used for anything else, so I've fixed it now